README.md

```diff
@@ -418,6 +404,12 @@ or to your ``pom.xml`` if you are using Maven
 
 ```
 
+Changelog
+-----------------
+
+We have removed the direct support to Renderers and Butter Knife libraries on Rosie to simplify its use and make easier its support. The sample is still using this libraries, so you can find an example of how to integrate this libraries in your project.
+
+
```

Comment:
@luis-ibanez  we are not using a changelog for this project we use the release information to notify the release changes if needed.